coalib/parsing/LineParser.py

```diff
@@ -78,6 +79,10 @@ def _parse(self, line):
                      [(section_override, key), ...], value, to_append (True if
                      append delimiter is found else False), comment
         """
+        if re.match('[^ ]#', line) is not None or\
+           re.match('#[^ ]', line) is not None:
```

Comment:
I think we should allow for lines with only a `#` in them: for example, if you're writing a long comment and want a paragraph break, this will be useful. Example:

```
# Line 1
#
# Line 2
```

But currently, this will be flagged.